docs/sources/enterprise/reporting.md

```diff
@@ -165,6 +165,8 @@ These options are available in the [configuration]({{< relref "../administration
 
 ```ini
 [reporting]
+# Use this option to enable or disable the reporting feature. When disabled, no reports are generated, and the UI is hidden. By default, reporting is enabled.
+enabled = true
```

Comment:
Can we use activate/deactivate or turn on/off? In the interests of inclusive language and all that - enable/disable mplies that disability is a less-desired or negative state. Thanks!